freemius/Freemius.php

```diff
@@ -251,6 +251,9 @@ public function MakeRequest($pCanonizedPath, $pMethod = 'GET', $pParams = array(
                     }
 
                     $opts[CURLOPT_RETURNTRANSFER] = true;
+                } else if ( 'GET' === $pMethod && ! empty( $pParams ) ) {

```

Comment:
Please take a look at the rest of the file :) We are not using WP coding standard, rather Freemius. So please adjust accordingly

```php
}
else if ('GET' === $pMethod && ! emoty($pParams)
{
    $pCanonizedPath = $this->AddQueryParams($pCanonizedPath, $pParams);
}
```

Also good to extract the `AddQueryParams` to the base abstract class (since we already have that).